### DISCUSSION TITLE
Problem while testing manim after installation

### DISCUSSION THREAD

**Discussion by haseebhijazi** [3b1b/manim#1418] [Q&A] [answered]
When I write: C:\Program Files\Manim\manim-master>python -m manim example_scenes.py SquareToCircle, in cmd, it shows: C:\Users\User\AppData\Local\Programs\Python\Python39\python.exe: No module named manim.
I also tried: C:\Program Files\Manim\manim-master>python -m manimgl example_scenes.py OpeningManimExample. 
It shows: C:\Users\User\AppData\Local\Programs\Python\Python39\python.exe: No module named manimgl
Please help me out.

**Comment by naveen521kk** [comment, marked as answer]
The python module here is `manimlib` and not `manimgl`. You can use `manimgl` command though. Or you should use `python -m manimlib`.
